src/api/instance.ts

```diff
@@ -1,10 +1,50 @@
-import axios from 'axios';
+import axios, { isAxiosError } from 'axios';
 import { BACKEND_ENDPOINT } from '@/constants/endpoint';
+import STORAGE_KEYS from '@/constants/storageKeys';
+import { ROUTER_PATHS } from '@/router';
+import ERROR_RESPONSES from '@/constants/errorMessages';
+import authAPI from './auth/apis';
 
 const baseInstance = axios.create({
   baseURL: BACKEND_ENDPOINT,
   timeout: 30000,
   withCredentials: true,
 });
 
+baseInstance.interceptors.request.use((config) => {
+  config.headers.accessToken = localStorage.getItem(STORAGE_KEYS.accessToken);
+
+  return config;
+});
```

Comment:
ask;
저도 잘 모르긴한데... 이렇게 하면 모든 요청에 accessToken 이 들어가고 없으면 null 이나 빈문자열이 들어갈 거 같은데 엑세스 토큰이 필요없는 곳엔 이렇게 해도 될까요??

```suggestion
baseInstance.interceptors.request.use((config) => {
  if (config.url !== '/api/oauth2/authorization/kakao') {
    config.headers.accessToken = localStorage.getItem(STORAGE_KEYS.accessToken);
  }
  return config;
});
```

Reply:
@easyhyun00 

음.. 인가 권한이 필요 없는 경우에는 `baseInstance` 를 사용하고, 필요한 경우에는 `authInstance` 를 사용하는 건 어떨까요??

```ts
/** 단순한 API 요청 클라이언트 */
export const baseInstance = axios.create({
  baseURL: BACKEND_ENDPOINT,
  timeout: 30000,
});

/** 사용자 인가 권한이 필요한 API 요청 클라이언트 */
export const authInstance = axios.create({
  baseURL: BACKEND_ENDPOINT,
  timeout: 30000,
});

authInstance.interceptors.request.use();
authInstance.interceptors.response.use();
```